### PR TITLE
CRN-1095 Inactive badge in events for teams and groups

### PR DESCRIPTION
**Pull request by gabiayako**
Jira Ticket: https://asaphub.atlassian.net/browse/CRN-1095

List View            |  Detail Page
:-------------------------:|:-------------------------:
list  | detail

Link to List View: https://2198.hub.asap.science/events/past?searchQuery=inactive
Link to Detail Page: https://2198.hub.asap.science/events/22868eed-c1ee-405e-b4bf-381a1c11fdc0

### CODE REVIEW

File: apps/crn-server/test/controllers/events.test.ts [hunk 1]
```diff
@@ -575,7 +591,19 @@ describe('Event controller', () => {
...
-      expect(result).toMatchObject(getEventResponse());
+
+      expect(result).toMatchObject({
+        ...getEventResponse(),
+        speakers: [
+          {
+            ...getEventSpeakerUser(),
+            team: {
+              ...getEventSpeakerUser().team,
+              inactiveSince: '2021-10-12T15:42:05Z',
```
Comment (peterstarling) [inline apps/crn-server/test/controllers/events.test.ts:602]: same as above, can you not add this to the fixture?

File: apps/crn-server/test/controllers/events.test.ts [hunk 2]
```diff
@@ -52,7 +52,23 @@ describe('Event controller', () => {
...
+      expect(result).toMatchObject({
+        items: [
+          {
+            ...getEventResponse(),
+            speakers: [
+              {
+                ...getEventSpeakerUser(),
+                team: {
+                  ...getEventSpeakerUser().team,
+                  inactiveSince: '2021-10-12T15:42:05Z',
```
[the remark this answers is not in the document]
Comment (peterstarling) [reply apps/crn-server/test/controllers/events.test.ts:64]: speaking of testing it individually - what happens when it comes back as null?

**Review by peterstarling — CHANGES_REQUESTED** [review]

File: apps/crn-server/test/controllers/events.test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (gabiayako) [reply apps/crn-server/test/controllers/events.test.ts]: I added it now here https://github.com/yldio/asap-hub/pull/2198/commits/9f8fdbdd23e072bd93ce1c6cb71eed5f7b845ee3 :)


The tests I added this `inactiveSince: '2021-10-12T15:42:05Z'` make use of this `getSquidexGraphqlClientMockServer` and it seems that when it comes back as undefined is being replaced by Instant (https://github.com/yldio/asap-hub/blob/CRN-1095-calendar-and-events-inactive-groups-list-and-detail-view/apps/crn-server/test/mocks/squidex-graphql-client-with-server.mock.ts#L41). I'm having trouble to understand where this replacing is coming from though. Is it something from createMockStore?

File: apps/crn-server/src/entities/event.ts [hunk 3]
```diff
@@ -94,18 +96,20 @@ export const parseGraphQLSpeakers = (
         team: {
           id: team.id,
           displayName: team.flatData.displayName ?? '',
+          inactiveSince: team.flatData.inactiveSince ?? undefined,
         },
-      };
+      } as EventSpeakerTeam;
```
[the remark this answers is not in the document]
Comment (gabiayako) [reply apps/crn-server/src/entities/event.ts]: yeah! that's right! I didn't notice before that changing the type of display name would give an error without the cast! Thanks!
I removed the casts in here https://github.com/yldio/asap-hub/pull/2198/commits/76dfe3673930db687e64d2ded4bc504894a34648

**Pull request merged**